src/test/java/com/google/sps/data/friend_map/FriendsMapCreationTest.java

```diff
@@ -48,85 +45,75 @@
   private UserNode userC;
   private UserNode userD;
   private UserNode userE;
-  private ImmutableSet<String> userAFriends;
-  private ImmutableSet<String> userBFriends;
-  private ImmutableSet<String> userCFriends;
-  private ImmutableSet<String> userDFriends;
-  private ImmutableSet<String> userEFriends;
-
-  @Before
-  public void setUp() {
-    userAFriends = ImmutableSet.of();
-    userBFriends = ImmutableSet.of();
-    userCFriends = ImmutableSet.of(USER_E_ID);
-    userDFriends = ImmutableSet.of(USER_E_ID);
-    userEFriends = ImmutableSet.of(USER_C_ID, USER_D_ID);
-    userA = new UserNode(USER_A_ID, userAFriends);
-    userB = new UserNode(USER_B_ID, userBFriends);
-    userC = new UserNode(USER_C_ID, userCFriends);
-    userD = new UserNode(USER_D_ID, userDFriends);
-    userE = new UserNode(USER_E_ID, userEFriends);
-  }
 
   /**
-  * Tests if graph is forming correctly when there is only one user with 0 friends.
+  * Tests if map is forming correctly when there is only one user with 0 friends.
   *
-  * Should result in a UserFriendsGraph with a map that has a single key that maps
+  * <p>Should result in a UserFriendsMap that has a single key that maps
   * to an empty friend set.
   */
   @Test
   public void oneUserGraphTest() {
+    userA = new UserNode(USER_A_ID, ImmutableSet.of());
+
     Set<UserNode> oneUserSet = new HashSet<>();
     oneUserSet.add(userA);
     UserFriendsMap oneUserFriendsMap = new UserFriendsMap(oneUserSet);
 
     Map<String, ImmutableSet<String>> expectedMap = new HashMap<>();
```

Comment:
```
ImmutableMap<String, ImmutableSet<String>> expectedMap =
    ImmutableMap.of(
        USER_A_ID, ImmutableSet.of()
    );
```

Reply:
Got it! I went ahead and did this for the expected maps for the other two tests as well.